docs/python/settings-reference.md

```diff
@@ -20,19 +20,53 @@ For general information about working with settings in VS Code, refer to [User a
-| logging.level| `error` | Specifies the level of logging to be performed by the extension.<br> The possible levels of logging, in increasing level of information provided, are `off`, `error`, `warn`, `info`, and `debug`.<br> When set to `off`, which is not recommended, basic information will still be shown such as startup information and commands run by the Python extension.<br> At the `error` level, basic information and errors will be shown.<br> At the `warn` level, basic, error, and warning information will be shown. At the `info` level, basic, error, warning, and additional information like method execution times and return values will be shown. At this time, the `debug` level doesn't display additional information. |
-| experiments.enabled | `true` |  Enables [A/B experiments in the Python extension](https://aka.ms/AAjvt9q). If enabled, you may be provided with proposed enhancements and/or features. |
+| terminal.launchArgs | `[]` | Launch arguments that are given to the Python interpreter when you run a file using commands such as **Python: Run Python File in Terminal**.<br> In the `launchArgs` list, each item is a top-level command-line element that's separated by a space (quoted values that contain spaces are a single top-level element and are thus one item in the list). <br> For example, for the arguments `--a --b --c {"value1" : 1, "value2" : 2}`, the list items should be `["--a", "--b", "--c", "{\"value1\" : 1, \"value2\" : 2}\""]`. <br> Note that VS Code ignores this setting when debugging because it instead uses arguments from your selected debugging configuration in `launch.json`. |
+| venvFolders | `[]` | Paths to folders where virtual environments are created. <br> Depending on the virtualization tool used, it can be the project itself: `${workspaceFolder}`, or separate folders for all virtual environments located side by side: `.\envs`, `~/.virtualenvs`, and so on. |
+
+## Debugger Settings
+
+### General debugging
+
+| Setting<br/>(python.debugpy.) | Default | Description | See also |
+| --- | --- | --- | --- |
+| debugJustMyCode | `true` | Specifies whether the debugger should only step through user-written code. Disabling allows you to step through library code as well. | [Debugging](/docs/python/debugging.md) |
+
+## Testing settings
+
+### General testing
+
+| Setting<br/>(python.testing.) | Default | Description | See also |
+| --- | --- | --- | --- |
+| autoTestDiscoverOnSaveEnabled | `true` | Specifies whether to enable or disable auto run test discovery when saving a test file. | [Testing](/docs/python/testing.md) |
+| cwd | null | Specifies an optional working directory for tests. | [Testing](/docs/python/testing.md) |
+| debugPort | `3000` | Port number used for debugging of unittest tests. | [Testing](/docs/python/testing.md) |
+| promptToConfigure | `true` | Specifies whether VS Code prompts to configure a test framework if potential tests are discovered. | [Testing](/docs/python/testing.md)  |
```

Comment:
we now have a new setting I forgot to mention in the meeting: https://github.com/microsoft/vscode-python/commit/08e228df23b3efbfa404a472a249d482bcab9ee4

Reply:
this one impacts autoTestDiscoverOnSaveEnabled as well as it might change the file type which refresh happens on

Reply:
Since this isn't a comprehensive setting list, I think I am going to leave this new setting out since we include `autoTestDiscoverOnSaveEnabled` which I assume to be more widely adopted. We should make sure we document this new setting in the testing docs when we update it later this iteration though.